### PR TITLE
DM-14846: Add configuration to fix build

### PR DESCRIPTION
**Pull request by SimonKrughoff**

### CODE REVIEW

File: ups/display_ginga.cfg [hunk 1]
```diff
@@ -1,4 +1,5 @@
 # -*- python -*-
+import lsst
```
Comment (PaulPrice) [inline ups/display_ginga.cfg]: Other files like this (e.g., in afw) use `import lsst.sconsUtils`.

**Review by PaulPrice — APPROVED** [review]
One suggestion.

File: ups/display_ginga.table [hunk 2]
```diff
@@ -1,2 +1,3 @@
 envPrepend(PYTHONPATH, ${PRODUCT_DIR}/python)
+setupRequired(base)
```
Comment (PaulPrice) [inline ups/display_ginga.table]: Given that you use in the interface defined in afw, why wouldn't you `setupRequired(afw)`?

Comment (SimonKrughoff) [reply ups/display_ginga.table]: Right.  I was trying to setup the minimum to build, but you are right that I should set up the packages needed to run.

File: ups/display_ginga.cfg (hunk 1 above)
Comment (SimonKrughoff) [reply ups/display_ginga.cfg]: OK.

**Pull request merged**